slides/slides.md

```diff
@@ -183,7 +183,7 @@ https://expo.dev/client
   - `npx create-expo-app NodeConfGame --template`
   - Pick <b>"Blank"</b> for JavaScript, or "Blank (TypeScript)"
   - `cd NodeConfGame` then  `npm i`
-  - `npx expo start --tunnel`
+  - `npx expo start --lan`
```

Comment:
Isn't lan the default? And [`--lan` isn't listed in the docs](https://docs.expo.dev/more/expo-cli/#develop), so if we make this change I think we should just remove the flag?

I thought `--tunnel` would be more robust with hotel wifi but you're right that the docs do say it requires ngrok - I don't remember installing ngrok but I guess I must have done so a long time ago.

Also if we're changing the recommendation here we should change the detail part in `./slides0/README.md` under `### 3. Start Expo` (both the command and the `[!NOTE]` underneath)

Reply:
> I don't remember installing ngrok, but I guess I must have done so a long time ago.

🤔 I don't seem to have ngrok installed (doing `ngrok` in a terminal gives "command not found: ngrok") and I don't have any ngrok-related global npm packages installed either. Does `--tunnel` work for you on a clean install?

Reply:
I don't mind --tunnel as a thing, however I'm not sure how the conference wifi would work with it, especially if there are quite a few tunnels opened at once.

If do we need them on the same WiFi if we're going to tunnel? I'd assume not since that's the point of tunnelling. If we use tunnel then we can probably remove the part about it needing to be on the same network.

Reply:
`--tunnel` worked for me, but required/prompted me to install the ngrok npm package to work.